vscode/src/commands.ts

```diff
@@ -2,6 +2,16 @@ import * as vscode from "vscode";
 
 import { createDocumentViewPanel } from "./webviews";
 
+import { posthog } from 'posthog-js'
+
+
+const isTelemetryEnabled = vscode.env.isTelemetryEnabled;
+
+
+if (isTelemetryEnabled) {
+  posthog.init('LeXA_J7NbIow0-mEejPwazN7WvZCj-mFKSvLL5oM4w0', { api_host: 'https://events.stencila.cloud' })
+}
+
```

Comment:
I suggest moving this to a new module `events.ts` and defining an `event` function that is conditional on `vscode.env.isTelemetryEnabled`.

```ts
export const event = isTelemetryEnabled
  ? (name: string, data?: any) => posthog.capture(name, data)
  : (name: string, data?: any) => {};
```

This will reduce the amount of boilerplate code at call sites:

```ts
 import { event } from "./events.ts"

event("vscode_preview");
```